FreeBSD's evdev has Linux's input API in <dev/evdev/input.h>.

--- a/headphoned.c
+++ b/headphoned.c
@@ -22,7 +22,11 @@
  */
 #include <errno.h>
 #include <fcntl.h>
+#ifdef __FreeBSD__
+#include <dev/evdev/input.h>
+#else
 #include <linux/input.h>
+#endif
 #include <stdio.h>
 #include <string.h>
 #include <sys/ioctl.h>
